### PR TITLE
webui: fix broken link for privacy policy

### LINKED ISSUE #3971: broken comments help (missing markdown link, broken privacy link)
This is how the comments help looks now:
bodhi

"Fedora-flavored Markdown" is no longer a link - it used to be a link, even though I don't know where exactly it pointed. Now, nobody can figure out what the Fedora enhancements are.

Privacy policy link is broken, you can see its actual value in the bottom left corner.

 bodhi-5.2.2

### PR DESCRIPTION
**Pull request by richardgreg**
Signed-off-by: Richard O. Gregory <richardgrecoson@gmail.com>

### CODE REVIEW

**Comment by mattiaverga** [comment]
I don't understand what are you trying to fix. Also, this change is wrong.

**Comment by richardgreg** [comment]
@mattiaverga 
>I don't understand what are you trying to fix.

Inside of an update's comment box, the link to `this privacy policy` in the help icon (top right) is broken.

>Also, this change is wrong.

Yeah I noticed that. I didn't get the chance try it in my local machine because I'm finding it difficult to log into the app from `localhost:6543` and interact with the comment box. Some things I've tried already is removing `stg` from the values of `openid.provider openid.url openid_template` in `development.ini.example`, but it's not working. I'm trying to open an account  in `https://admin.stg.fedoraproject.org/accounts` but I have not received a confirmation email yet. I'm not sure how long it should be taking. Do you have any suggestions?

**Comment by mattiaverga** [comment]
Now I see... I've never noticed that broken link. It's really an awful case of mixing multiple quotes and double quotes.
I think the best (or the only) solution is to assign `request.registry.settings.get('privacy_link')` to a variable and use that inside the href property.

About opening an account into staging fas, it's been a long time I did... not sure how long it takes to receive the confirmation email, but I suppose not a lot of time.

**Comment by richardgreg** [comment]
@mattiaverga okay.
Also wrapping the strings in docstrings seems to not break anything, but so does the current implementation. But in this case, I notice that using single or double quotes interchangeably around `href` no longer raises any error.

No matter the approach, the main problem for me now is that I can't test the functionality because `Legal` and `Privacy Policy` links don't show up when running Bodhi in local machine. Is this a normal feature?

**Comment by mattiaverga** [comment]
> 
> No matter the approach, the main problem for me now is that I can't test the functionality because `Legal` and `Privacy Policy` links don't show up when running Bodhi in local machine. Is this a normal feature?

You need to add `privacy_link = whatever` in `development.ini` under `[app:main]` section.

File: bodhi/server/templates/update.html [hunk 1]
```diff
@@ -219,8 +219,9 @@ <h4>How to install</h4>
                   <div class="col pr-0">
                     <%
                       help_content = "Comments support Fedora-Flavored Markdown"
+                      privacy_link = request.registry.settings.get('privacy_link')
```
Comment (mattiaverga) [inline bodhi/server/templates/update.html]: I think it's better to assign the variable only after having checked `request.registry.settings.get('privacy_link')` is set. I would move this line inside the if statement below.

**Review by cverna — APPROVED** [review]
LGTM

**Comment by richardgreg** [comment]
@cverna I opened this PR before #3971 was filed. I see that the 'fedora-flavoured markdown' help not popping up is also part of the issue. I talked about the pop-up when starting work on #3209. Is there any particular reason why the feature was retracted? If there is no problem I can work-on putting it back.

**Comment by mattiaverga** [comment]
I would prefer to get rid of that modal popup and link that help to the doc page you are working on in #3985 (just opening it in a new window/tab). That will avoid us to maintain the help text in two different places.

If @cverna agrees to that, I think we can merge this PR as is and then @richardgreg can change #3985 to add the missing link.

**Comment by cverna** [comment]
> I would prefer to get rid of that modal popup and link that help to the doc page you are working on in #3985 (just opening it in a new window/tab). That will avoid us to maintain the help text in two different places.

:+1: 

> 
> If @cverna agrees to that, I think we can merge this PR as is and then @richardgreg can change #3985 to add the missing link.

Sounds good to me

**Pull request merged**